frontend/components/AuthenticationFormWrapper/AuthenticationFormWrapper.jsx

```diff
@@ -12,7 +12,7 @@ class AuthenticationFormWrapper extends Component {
 
     return (
       <div className={baseClass}>
-        <img alt="Kolide text logo" src="/assets/images/kolide-logo-text.svg" className={`${baseClass}__logo`} />
+        <img alt="Kolide vertical logo" src="/assets/images/kolide-logo-vertical.svg" className={`${baseClass}__logo`} />
```

Comment:
The alt should just be `Kolide` or maybe `Kolide Fleet`

Reply:
Changed